lib/datapacksutils.js

```diff
@@ -3042,8 +3042,115 @@ DataPacksUtils.prototype.loadSalesforceMetadata = async function() {
 
 var folderOnlyMetadata = [ "aura", "layouts", "lwc", "staticresources" ]
 
-DataPacksUtils.prototype.retrieveSalesforce = async function(jobInfo, listOnly) {
+DataPacksUtils.prototype.getAdditionalSFMetadataDependencies = async function(jobInfo, sfFolderPath){
+    let vlocityDataPackTypes = [],
+        salesforceKeys = [],
+        salesforceTypes = [];
+
+    vlocityDataPackTypes = Object.keys(await this.vlocity.utilityservice.getVlocityDataPackConfigurations());
+    
+    salesforceKeys = jobInfo.specificManifestKeys.filter(item => {
+        return vlocityDataPackTypes.indexOf(item.split('/')[0]) === -1;
+    });
+
+    salesforceTypes = Array.from(new Set(
+        salesforceKeys.map(item => {
+            return item.split('/')[0];
+        })
+    ));
+
+    if (salesforceTypes.indexOf('classes') !== 0) {
+        var allApexClassData = await this.vlocity.utilityservice.fetchAllRecords('SELECT Id,Name FROM ApexClass');
```

Comment:
Here you can change it to where NamespacePrefix = null - fine for this feature to only work in managed package.